common/views/components/PrototypeSearchFilters/PrototypeSearchFiltersMobile.tsx

```diff
@@ -51,6 +52,7 @@ const CloseFiltersButton = styled(Space).attrs({
   className: classNames({
     'plain-button': true,
   }),
+  'aria-label': 'close filter button',
```

Comment:
Should this be put into the labels file? I'd also use natural language, e.g. "Close filters"

Reply:
ahh i didnt see this one ... ill create a new pr for this.. thanks